File: pinyougou-manager-web/src/main/java/com/pinyougou/manager/controller/SellerController.java
package com.pinyougou.manager.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.pinyougou.entity.ResponseResult;
import com.pinyougou.pojo.TbSeller;
import com.pinyougou.sellergoods.service.SellerService;
import org.springframework.web.bind.annotation.*;

import java.util.List;
import java.util.Map;

/**
 * @program: pinyougou-all
 * @description: 商家审核的
 * @author: YF
 * @create: 2018-09-29 23:42
 **/
@RestController
@RequestMapping("/seller")
public class SellerController {
    @Reference
    private SellerService sellerService;

    /** 
    * @Description: 查找所有的
    * @Param: [] 
    * @return: java.util.List<com.pinyougou.pojo.TbSeller> 
    * @Author: Yin 
    * @Date: 2018/9/30 
    */ 
    @RequestMapping("/findAll")
    public List<TbSeller> findAll(){
        return sellerService.findAll();
    }

    
    /** 
    * @Description: 商家的审核的查看的条件分页显示
    * @Param: [page, rows, seller] 
    * @return: java.util.Map<java.lang.String,java.lang.Object> 
    * @Author: Yin 
    * @Date: 2018/9/30 
    */ 
    @RequestMapping("/findByCondition")
    public Map<String,Object> findByCondition(@RequestParam(required = false,defaultValue = "1") int page,
                                              @RequestParam(required = false,defaultValue = "1") int rows,
                                              @RequestBody TbSeller seller){
        return sellerService.findByCondition(page,rows,seller);
    }


    /** 
    * @Description: 根据id去进行查找到整个商家的详情，也就是为了进行修改所做的操作
    * @Param: [sellerId] 
    * @return: com.pinyougou.pojo.TbSeller 
    * @Author: Yin 
    * @Date: 2018/9/30 
    */ 
    @RequestMapping("/findOne")
    @ResponseBody
    public TbSeller findOne(String sellerId){
        return sellerService.findOne(sellerId);
    }

    /** 
    * @Description: 修改商家的审核状态
    * @Param: [sellerId, status]
    * @return: com.pinyougou.entity.ResponseResult<com.pinyougou.pojo.TbSeller> 
    * @Author: Yin 
    * @Date: 2018/9/30 
    */ 
    @RequestMapping("/updateStatus")
    public ResponseResult<TbSeller> updateStutas(String sellerId,String status){
        try {
            sellerService.updateStutas(sellerId,status);
            return ResponseResult.success("审核成功，请审核下一个");
        }catch (Exception e){
            e.printStackTrace();
            return ResponseResult.error("审核错误，请重新审核");
        }
    }
    
    
    
    
}

File: pinyougou-manager-web/src/main/webapp/js/controller/sellerController.js
app.controller('sellerController',function ($scope,$controller,sellerService) {
    //这就相当于继承了父类的操作
    $controller('baseController',{$scope:$scope});//继承

    //查找到所有的
    $scope.findAll=function () {
        sellerService.findAll().success(function (response) {
            $scope.list=response;
        })
    };

    //条件查询
    $scope.searchEntity={};
    $scope.findByCondition=function (page,rows) {
        sellerService.findByCondition(page,rows,$scope.searchEntity).success(function (response) {
            //更新数据列表
            $scope.paginationConf.totalItems=response.total;//总记录数
            $scope.list=response.rows;//给列表变量赋值
        })
    };

    //根据id去进行查找
    $scope.findById=function (sellerId) {
        sellerService.findById(sellerId).success(function (response) {
            $scope.entity=response;
        })
    };

    //修改状态
    $scope.updateStatus=function (sellerId,status) {
        sellerService.updateStatus(sellerId,status).success(function (response) {
            if (response.status==0){//修改成功
                //刷新列表
                $scope.reloadList();
            }else{
                //失败那就弹窗说明
                alert(response.message);
            }
        })
    };


})